assets/src/web-stories-block/block/api/useStoryApi.js

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo, useReducer } from 'react';
+import queryString from 'query-string';
+
+/**
+ * Internal dependencies
+ */
+import {
+  STORY_STATUSES,
+  STORY_SORT_OPTIONS,
+  ORDER_BY_SORT,
+  STORIES_PER_REQUEST,
+  STORY_STATUS,
+} from '../../../dashboard/constants';
```

Comment:
Those constants are very specific to the dashboard.

Let's use our own constants for the block.

Reply:
yes, I agree though in the story picker modal we have provided the sort options which are identical to the dashboard also components like the sort menu and InfiniteScroller provide updated values that can be used with the fetch API. Hence needed to keep this.

Though I think we can remove the `STORY_STATUS` constants as we will only be using published stories. Or we can move these to over constants JS. 🤔

Reply:
Yeah let's use our own `constants.js` file where possible.

`STORIES_PER_REQUEST` seems like  something we can tweak here as well, no?

Reply:
`STORIES_PER_REQUEST` is set to `24` do we want to set it to some other value?

Reply:
Maybe. Let's define `STORIES_PER_REQUEST` in our custom `constants.js` file